pages.pt-BR/windows/whoami.md

```diff
@@ -0,0 +1,24 @@
+# whoami
+
+> Mostra detalhes sobre o usuário atual.
+> Mais informações: <https://docs.microsoft.com/en-us/windows-server/administration/windows-commands/whoami>.
+
+- Mostra o nome de usuário do usário atual:
```

Comment:
```suggestion
Mostra o nome de usuário do usuário atual:
```

Reply:
@schneiderl O que achas de utilizar **login** ou **username** mesmo ao invés de nome de usuário? Sei que estão em inglês mas são termos bem aceitos na comunidade brasileira. Além disso ficaria consistente com o id de logon do último exemplo.

Reply:
Faz sentido. Vou fazer a alteração.

Reply:
Careful guys, this change removed the `-` at the start of the line. (How did Travis not complain about this?...)

Reply:
@waldyrious Fixed

Reply:
@waldyrious tldr-lint does not check pages of all languages:

https://github.com/tldr-pages/tldr/blob/77c60fc861c038d12082367c27851391a23e4e8b/package.json#L14